ginivision/src/main/java/net/gini/android/vision/analysis/AnalysisActivity.java

```diff
@@ -159,6 +160,8 @@ public void hideError() {
     public void noExtractionsFound() {
         final Intent noResultsActivity = new Intent(this, NoResultsActivity.class);
         startActivity(noResultsActivity);
+        setResult(Activity.RESULT_CANCELED);
```

Comment:
I think it's clearer if we set an own result code here, like `RESULT_NO_EXTRACTIONS`.